frontend/app/(defaultLayout)/Troubleshooting/[device]/components/NavBar.tsx

```diff
@@ -81,30 +77,15 @@ export function NavBar({
                   flex="1 2"
                   overflowX="auto"
                >
-                  <NavTabs
-                     overflowX="auto"
-                     flexGrow="1"
-                     paddingInline={{ base: 0, sm: 2 }}
-                     deviceGuideUrl={deviceGuideUrl}
-                     devicePartsUrl={devicePartsUrl}
-                  />
+                  <NavTabs deviceUrls={deviceUrls} />
                </Box>
             </Flex>
          </Flex>
       </Flex>
    );
 }
 
-type NavTabsProps = {
-   deviceGuideUrl?: string;
-   devicePartsUrl?: string;
-};
-
-function NavTabs({
-   devicePartsUrl,
-   deviceGuideUrl,
-   ...props
-}: NavTabsProps & FlexProps) {
+function NavTabs({ deviceUrls }: any) {
```

Comment:
We already have the exported type in `useTroubleShootingProbelmsProps`, could we use that type here instead of `any`?

Reply:
I tried that, but kept getting type errors that I couldn't resolve.

Reply:
Figured it out 🎉 https://github.com/iFixit/react-commerce/pull/2223/commits/af778ecaf327c5467f9ccfb17899c2c86dfd5b18